site/src/modules/workspaces/WorkspaceTiming/StagesChart.tsx

```diff
@@ -167,28 +83,26 @@ export const StagesChart: FC<StagesChartProps> = ({
 	);
 	const totalTime = calcDuration(totalRange);
 	const [ticks, scale] = makeTicks(totalTime);
+	const sections = Array.from(new Set(timings.map((t) => t.stage.section)));
```

Comment:
Please receive a review from someone in the frontend group for these changes.
I don't know enough to approve them.

Reply:
Sure thing, I just want to check the BE first.